src/workers/VolumeLoaderContext.ts

```diff
@@ -193,6 +207,14 @@ class VolumeLoaderContext {
     this.activeLoader = new WorkerLoader(this.activeLoaderId, this.workerHandle);
     return this.activeLoader;
   }
+
+  setThrottleChannelData(throttle: boolean): void {
+    this.workerHandle.setThrottleChannelData(throttle);
+  }
+
+  getActiveLoader(): WorkerLoader | undefined {
+    return this.activeLoader;
+  }
```

Comment:
This class is designed around the possibility of supporting more than one "active" loader at a time in the future. This method wouldn't make sense in that world, but removing it and replacing whatever depends on it with something that can live with multiple loaders should be about as hard as anything else we'd need to do to support that anyways.

Reply:
View3d.setupGui sets up a tweakpane handler that calls `      this.loaderContext?.getActiveLoader()?.updateFetchOptions({...
`

Reply:
(just needed some way to push these values through from the tweakpane layer)